docs/source/architecture/rest_api.rst

```diff
@@ -2,313 +2,47 @@
 REST API
 ********
 
-*Hyperledger Sawtooth* provides a pragmatic RESTish API for clients to interact
-with a validator using common HTTP/JSON standards. It is an entirely separate
-process, which once running, allows transactions to be submitted and blocks to
-be read with a common language-neutral interface. As the validator is redesigned
-and improved the REST API will grow with it, providing a consistent interface
-that meets the needs of application developers into the future.
-
-With that focus on application developers, the REST API treats the validator
-mostly as a
-black box, submitting transactions and fetching the results. It is not the tool
-for all validator communication. For example, it is not used by Transaction
-Processors to communicate with a validator, or by one validator to talk to other
-validators. For these and similar use cases, there is a more efficient and
-robust, if somewhat more complicated, ZMQ/Protobuf interface.
-
-
-Open API Specification
-======================
-
-The REST API is comprehensively documented using the
-`OpenAPI specification <http://swagger.io/specification/>`_ (fka Swagger),
-formatted as a YAML file. This documentation provides a single source of truth
-that completely documents every implemented aspect of the API, is both human and
-machine readable, and can be compiled by a variety of toolsets (including for
-this Sphinx-based document). The spec file itself can be found
-`here <https://github.com/hyperledger/sawtooth-core/blob/master/rest_api/openapi.yaml>`_.
-
-
-HTTP Status Codes
+Hyperledger Sawtooth provides a REST API (see the :doc:`../rest_api`) that
+allows clients to interact with a validator using common HTTP/JSON standards.
+It is a pragmatic RESTish API that provides a language-neutral interface for
```

Comment:
What is the motivation for calling this RESTish? Is it accurate enough to call it RESTful?

Reply:
The original doc said "RESTish".  However, I can't discern a difference between these two terms. Also, Peter just said it's OK to change it to "RESTful". So I shall make this change.